src/model/element/entity.js

```diff
@@ -177,6 +177,25 @@ class Entity extends Element {
 
     return entity;
   }
+
+  toIndraTemplate(){
+    let name = this.name();
+
+    // if the name is not set it means that there is no info for the entity
+    // so that entity must not be included
+    if ( !name ) {
+      return null;
+    }
+
+    let xref = _.pick(this.association(), ['id', 'namespace']);
+    let validNamespaces = ['hgnc', 'chebi'];
```

Comment:
- Case-sensitive

Reply:
Yes but the namespaces I have inspected so far was in full lowercases. I am able to inspect 'chebi' and 'ncbi' so far.

Reply:
Is HGNC in full uppercases in factoid?

Reply:
You're right, the association/grounding-service namespace is lower case ('chebi', 'ncbi'). My confusion is why 'hgnc' is within `validNamespaces`, since the grounding-service doesn't support that datasource?

Reply:
Looks like I misunderstood something and was thinking that ``hgnc`` is supported in a way. 

Then what should be in the valid namespaces now? I remember in the past we talked about hgnc and chebi.

Reply:
Can just put array in xref and let your indra.js module pick off what it needs.

Reply:
I>You're right, the association/grounding-service namespace is lower case ('chebi', 'ncbi'). My confusion is why 'hgnc' is within validNamespaces, since the grounding-service doesn't support that datasource?

I guess my main confusion is resolved in another part of conversation. As I mentioned I was thinkg that hgnc would be one of the namespaces but now I understand that it is not.

> Can just put array in xref and let your indra.js module pick off what it needs.

Yes I can do so if we rename the ``toIndraTemplate()`` functions as ``toTemplate()`` and make them more generic as you suggested.

In that case I consider including id, namespace, name, dbXrefs, parentId in the return value of ``entry.toTemplate()`` to make it more generic. Let me know if you think some other features should be included as well.

Reply:
Maybe `toSearchTemplate` or something a little more specific. That's it I think.